source/Keystone/index.rst

```diff
@@ -0,0 +1,8 @@
+Keystone
```

Comment:
Lets rename this to authentication as users don't need to know it's called auth, and IMO it should be under basics
Then we can move the clouds.yaml documentation into this section too